resources/lang/es.json

```diff
@@ -16,6 +16,7 @@
     "Regards": "Saludos",
     "Whoops!": "¡Ups!",
     "Hello!": "¡Hola!",
+    "All rights reserved.": "Todos los derechos resevados",
```

Comment:
It's already defined https://github.com/Laraveles/spanish/blob/master/resources/lang/es.json#L32